config/routes.rb

```diff
@@ -65,6 +65,9 @@
 
   resources :communities, only: [:new, :create]
 
+
+  devise_for :people, only: :omniauth_callbacks, controllers: { omniauth_callbacks: "sessions"}
```

Comment:
To my understanding this doesn't have any major effect. The path for the omniauth callback is fetched from `person_omniauth_authorize_path` method, so I guess from now on that method returns a path without any locale in it?

Anyway, I tested this and FB auth seems to be working normally.